GW2EIEvtcParser/EncounterLogic/Convergences/MountBalriorConvergenceInstance.cs

```diff
@@ -0,0 +1,170 @@
+        Segment start75 = hpUpdates.FirstOrDefault(x => x.Value < 75.0 && x.Value != 0 && x.Start > end75.End);
+        Segment end50 = hpUpdates.FirstOrDefault(x => x.Value < 50.0 && x.Value != 0);
+        Segment start50 = hpUpdates.FirstOrDefault(x => x.Value < 50.0 && x.Value != 0 && x.Start > end50.End);
+        Segment end25 = hpUpdates.FirstOrDefault(x => x.Value < 25.0 && x.Value != 0);
+        Segment final = hpUpdates.FirstOrDefault(x => x.Value < 25.0 && x.Start > end25.End);
+
+        // 100-75, Warclaw, 75-50, Warclaw, 50-25, Warclaw, 25-0
+        PhaseData phase1 = new(start.Start, Math.Min(end75.Start, log.FightData.FightEnd), "Phase 1");
+        PhaseData phase2 = new(start75.Start, Math.Min(end50.Start, log.FightData.FightEnd), "Phase 2");
+        PhaseData phase3 = new(start50.Start, Math.Min(end25.Start, log.FightData.FightEnd), "Phase 3");
+        PhaseData phase4 = new(final.Start, Math.Min(target.AgentItem.LastAware, log.FightData.FightEnd), "Phase 4");
+        PhaseData warclaw1 = new(end75.Start, Math.Min(start75.Start, log.FightData.FightEnd), "Warclaw 1");
+        PhaseData warclaw2 = new(end50.Start, Math.Min(start50.Start, log.FightData.FightEnd), "Warclaw 2");
+        PhaseData warclaw3 = new(end25.Start, Math.Min(final.Start, log.FightData.FightEnd), "Warclaw 3");
+
+        phase1.AddTarget(target, log);
+        phase2.AddTarget(target, log);
+        phase3.AddTarget(target, log);
+        phase4.AddTarget(target, log);
+        warclaw1.AddTarget(target, log);
+        warclaw2.AddTarget(target, log);
+        warclaw3.AddTarget(target, log);
+        phase1.CanBeASubPhaseOf(fullPhase);
+        phase2.CanBeASubPhaseOf(fullPhase);
+        phase3.CanBeASubPhaseOf(fullPhase);
+        phase4.CanBeASubPhaseOf(fullPhase);
+        warclaw1.CanBeASubPhaseOf(fullPhase);
+        warclaw2.CanBeASubPhaseOf(fullPhase);
+        warclaw3.CanBeASubPhaseOf(fullPhase);
+
+        phases.AddRange([phase1, phase2, phase3, phase4, warclaw1, warclaw2, warclaw3]);
+
+        return phases;
+    }
+
+    internal override void CheckSuccess(CombatData combatData, AgentData agentData, FightData fightData, IReadOnlyCollection<AgentItem> playerAgents)
+    {
+        base.CheckSuccess(combatData, agentData, fightData, playerAgents);
+
+        RewardEvent? reward = combatData.GetRewardEvents().FirstOrDefault(x => 
+            x.RewardType == RewardTypes.ConvergenceReward1 ||
+            x.RewardType == RewardTypes.ConvergenceReward2);
+        if (reward != null)
+        {
+            fightData.SetSuccess(true, reward.Time);
+        }
+    }
+
+    internal override void EIEvtcParse(ulong gw2Build, EvtcVersionEvent evtcVersion, FightData fightData, AgentData agentData, List<CombatItem> combatData, IReadOnlyDictionary<uint, ExtensionHandler> extensions)
+    {
+        AgentItem? ura = agentData.GetNPCsByID(TargetID.UraTheSteamshriekerConv).FirstOrDefault();
+        ura?.OverrideName("Ura, the Steamshrieker");
```

Comment:
Should be done after the EIEvtcParse, on SingleActor, AgentItems are what we read from arc directly, so when it is possible to apply the override on EI elements, that should be the prefered option.